Solutions/Corvus.Identity.Azure/Corvus/Identity/ClientAuthentication/Azure/Internal/AzureTokenCredentialSourceFromConfiguration.cs

```diff
@@ -0,0 +1,98 @@
+﻿// <copyright file="AzureTokenCredentialSourceFromConfiguration.cs" company="Endjin Limited">
+// Copyright (c) Endjin Limited. All rights reserved.
+// </copyright>
+
+namespace Corvus.Identity.ClientAuthentication.Azure.Internal
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    using global::Azure;
+    using global::Azure.Core;
+    using global::Azure.Identity;
+    using global::Azure.Security.KeyVault.Secrets;
+
+    /// <summary>
+    /// Implementation of <see cref="IAzureTokenCredentialSourceFromDynamicConfiguration"/>.
```

Comment:
What would you have it say? If it weren't for the tool-imposed requirement that everything have some docs, I'd be inclined to put nothing at all here, because everything interesting about this class is (or at least should be) in the docs for the interface. So this class's entire purpose is to be the one and only implementation of that interface. I don't like it when we just dup documentation from the interface into the class, and I don't know what useful information I could put here that wouldn't be just as useful in the interface docs.

Reply:
So I think your point is that this will be the only implementation of `IAzureTokenCredentialSourceFromDynamicConfiguration` we'd ever expect to see (barring test mocks/fakes)? In which case, that's fair enough.